src/include/iscsi-client/dialogs.rb

```diff
@@ -272,13 +272,13 @@ def initialize_iscsi_client_dialogs(include_target)
               _("No Authentication"),
               true,
               VBox(
-                Left(Label(_("Incoming Authentication"))),
+                Left(Label(_("Outgoing Authentication"))),
                 HBox(
                   HWeight(2, TextEntry(Id(:user_in), _("Username"))),
```

Comment:
@izhu, what @cwh42 means is that the, with your changes, the label "incoming authentication" relates to the text entry called ":user_out" and the password entry called ":pass_out". Quite inconsistent.

Changing the labels doesn't seem like the right approach, we should fix the issue in the lowest layers, where the values are really used.